### PR TITLE
Playbooks addition to existing Prisma cloud solution

### PR DESCRIPTION
**Pull request by manishkumar1991**
Required items, please complete
   
   Change(s):
   - Addition of new playbooks
   - Security posture enrichment
   - security alert remediation

   Reason for Change(s):
   - New Soar content added in Prisma cloud solution

   Version Updated:
   - N/A

   Testing Completed:
   - Completed

   Checked that the validations are passing and have addressed any issues that are present:
   - Passed

### CODE REVIEW

**Comment by manishkumar1991** [comment]
> @microsoft-github-policy-service agree company="Microsoft"

@microsoft-github-policy-service agree company="Microsoft"

File: Solutions/PaloAltoPrismaCloud/Playbooks/PrismaCloudCSPMPlaybooks/PrismaCloudCSPM-Remediation/readme.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# PrismaCloudCSPM-Enrichment Info Playbook
```
Comment (rahul0216) [inline Solutions/PaloAltoPrismaCloud/Playbooks/PrismaCloudCSPMPlaybooks/PrismaCloudCSPM-Remediation/readme.md:1]: 1. Same spelling mistakes as other readme files.
2. Update the reference to PrismaCloudCSPM-Enrichment playbook to PrismaCloudCSPM-Remediation playbook.

**Review by rahul0216 — COMMENTED** [review]
1 .Please address the inline comments.
2. Please upload playbbok screenshot in dark theme as well.
Mostly looks good.

File: Solutions/PaloAltoPrismaCloud/Playbooks/PrismaCloudCSPMPlaybooks/PrismaCloudCSPM-Enrichment/readme.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+ 1. Fetches the list of assets from incident entites .
+ 2. Make the APi call to get the latest Compliance security posture/details from cloud console and update the same incidents comments with result.
+ 3. Note : Asset column for which the security posture to be enriched should be mapped with hostname entity while creating analyics rule. 
+
+
+### Prerequisites 
+"1. PrismaCloudCSPM Custom Connector needs to be deployed prior to the deployment of this playbook under the same subscription.",
+"2. API Key and User ID . To get this, login into your Prisma cloud instance dashboard and navigate to Settings --> Access Control --> Access Keys --> Add",
+"3. [Important step]Store the API secret key in Key vault then provide the keyvault name and key name of the stored secret during deployment"
+
```
[the remark this answers is not in the document]
Comment (manishkumar1991) [reply Solutions/PaloAltoPrismaCloud/Playbooks/PrismaCloudCSPMPlaybooks/PrismaCloudCSPM-Enrichment/readme.md:13]: Done

**Review by rahul0216 — APPROVED** [review]
Chages look good. Ready to merge.

**Pull request merged**